internal/log/log.go

```diff
@@ -13,51 +12,29 @@ import (
 var Logger *zap.SugaredLogger
 
 func init() {
-	config := zap.NewDevelopmentConfig()
-	config.DisableStacktrace = true
-	config.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
-	logger, err := config.Build()
-	if err != nil {
-		// notest
-		log.Fatalln("failed to initialise logger")
-	}
-	Logger = logger.Sugar()
+	SetGlobalLogger("info")
 }
 
-// ReplaceGlobalLogger replace the logger and inject it globally
-func ReplaceGlobalLogger(enableJsonOutput bool, verbosityLevel string, enableColorEncoder bool) error {
+// SetGlobalLogger replace the logger and inject it globally
+func SetGlobalLogger(verbosityLevel string) error {
 	config := zap.NewProductionConfig()
-
-	// Colour coding
-	if enableColorEncoder {
-		config.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
-	} else {
-		config.EncoderConfig.EncodeLevel = zapcore.CapitalLevelEncoder
-	}
+	config.EncoderConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	config.Encoding = "console"
 
 	// Timestamp format (ISO8601) and time zone (UTC)
-	config.EncoderConfig.EncodeTime = zapcore.TimeEncoder(func(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
-		// notest
+	config.EncoderConfig.EncodeTime = func(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
 		enc.AppendString(t.UTC().Format("2006-01-02T15:04:05Z0700"))
-	})
-
-	// Output encoding
-	config.Encoding = "console"
-	if enableJsonOutput {
-		config.Encoding = "json"
```

Comment:
We are removing the JSON format for logs?

Reply:
In the past whenever I had to sieve through logs I would redirect the stdout to a file. So I didn't think there was a need for it.

Reply:
It was added for an external contributor that basically wanted to display logs using metrics directly and some parsing tools, Is a plus, that if not a big deal, I will prefer to keep

Reply:
For now, I don't think it is required. We can revisit this in the future.

Reply:
@adriantpaez can we resolve the conversation?